Sprint-2/implement/contains.js

```diff
@@ -1,3 +1,8 @@
-function contains() {}
+function contains(object, property) {
+    if (typeof object !== 'object' || object === null || Array.isArray(object)) {
+        return false;
+    }
+    return object.hasOwnProperty(property);
```

Comment:
The `hasOwnProperty` function works here.  You could also write:
`return object[property] !== undefined;`